Frontend/app/src/main/java/io/happybugs/happybugs/MainActivity.java

```diff
@@ -0,0 +1,125 @@
+package io.happybugs.happybugs;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.design.widget.BottomNavigationView;
+import android.support.design.widget.FloatingActionButton;
+import android.support.design.widget.Snackbar;
+import android.view.View;
+import android.support.design.widget.NavigationView;
+import android.support.v4.view.GravityCompat;
+import android.support.v4.widget.DrawerLayout;
+import android.support.v7.app.ActionBarDrawerToggle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.widget.TextView;
+
+import io.happybugs.happybugs.R;
```

Comment:
Can we come up with a better name than R?

Reply:
I think R is generated android studio by default.